src/k6.ts

```diff
@@ -459,6 +459,62 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "scale",
+      description: "Scale a running test",
+      subcommands: [
```

Comment:
I missed this one 🤦‍♂️ 
```suggestion
      options: [
```